### PR TITLE
NO-ISSUE: Update OWNERS

### PR DESCRIPTION
**Pull request by honza**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,11 +1,11 @@
 approvers:
-  - asalkeld
   - cgwalters
   - dtantsur
   - elfosardo
-  - hardys
+  - honza
+  - hroy
```
Comment (elfosardo) [inline OWNERS:6]: this needs to be changed to hroyrh

Comment (honza) [reply OWNERS]: Done

**Pull request merged**